static/js/pivotal.js

```diff
@@ -8,6 +8,12 @@
     }
   };
 
+  // register common error handler on Ajax errors / failures
+  function logError(err) {
```

Comment:
thought of that as well initially as well, but decided against it somehow (based on second-guessing myself). will do so!